snuba/settings/__init__.py

```diff
@@ -23,6 +23,16 @@
 
 ADMIN_AUTH_PROVIDER = "NOOP"
 
+# Migrations Groups that are allowed to be managed
+# in the snuba admin tool.
+ADMIN_ALLOWED_MIGRATION_GROUPS = {
+    "system",
+    "generic_metrics",
+    "profiles",
+    "functions",
+    "replays",
+}
+
```

Comment:
Any reason for not using the existing SKIPPED_MIGRATIONS_GROUPS
https://github.com/getsentry/snuba/blob/master/snuba/settings/__init__.py#L206
Which is the flag that makes the migration tool ignore migration groups?

Or is this meant to serve a different purpose? 
Specifically how are we dealing with these use cases:
- a group that has not been vetted yet. That should be disabled to the tool entirely whether it is CLI or through admin.
- a group whose cluster is not ready. The dataset was merged by the cluster is not provisioned yet. Should these migrations be visible in the tool? Visible and disabled ?

Reply:
> Any reason for not using the existing `SKIPPED_MIGRATIONS_GROUPS`

I wanted to have a list that you have to explicitly define which groups are allowed, vs implicitly assuming all that are not in the set are allowed. I think by using `get_active_migration_groups` I'm actually using the `SKIPPED_MIGRATIONS_GROUPS` as well, but I wanted to limit more than just that. 

> Or is this meant to serve a different purpose?

`ADMIN_ALLOWED_MIGRATION_GROUPS` is meant to serve as a safeguard for the admin tooling specifically right now for datasets who we haven't determined are safe to run. (thinking of errors in particular). I think we will have to have a more robust system for permissions going forward.

> a group that has not been vetted yet. That should be disabled to the tool entirely whether it is CLI or through admin.

This is kind of what `ADMIN_ALLOWED_MIGRATION_GROUPS` is used for except only specifically for admin. I think if migrations are not to be run at all, via admin or cli, then they should go in `SKIPPED_MIGRATION_GROUPS`.

> a group whose cluster is not ready. The dataset was merged by the cluster is not provisioned yet. Should these migrations be visible in the tool? Visible and disabled ?

I think this is taken care of by `SKIPPED_MIGRATIONS_GROUPS`